plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties

```diff
@@ -26,6 +27,9 @@ deploy.assembly.test.source.remove.job=Removing test Java source folder from web
 CREATE_AS_MAVEN_PROJECT=Create as Maven project
 MAVEN_PROJECT_COORDINATES=Maven project coordinates
 GROUP_ID=Group ID:
+FLEX_JAR_SAMPLE_TEMPLATE=Sample template:
+FLEX_JAR_SIMPLE_TEMPLATE=simple
```

Comment:
I'm not sure "simple" is the right description here; though I'm not sure what is. Maybe Minimal? No Dependencies?

Ideally I'd like to replace this template with a Jetty based template, but I'm not sure we have time to do that.

Reply:
No Web Framework?

It's easy to add a Jetty template, BTW.